introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Dean and I've come a long way from knowing nothing about software development to becoming nearly proficient to be deemed an entry level developer. I still need to round off the rough edges of my skills and hopefully by the end of the year I'll be good enough to produce professional grade work.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.